include/ensmallen_bits/cmaes/transformation_policies/boundary_box_constraint.hpp

```diff
@@ -0,0 +1,166 @@
+/**
+ * @file boundary_box_constraint.hpp
+ * @author Suvarsha Chennareddy
+ *
+ * Boundary Box Transformation.
+ *
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+namespace ens {
+  /**
```

Comment:
```suggestion
namespace ens {

  /**
```
Some more style pedantry.  Also, it seems like this comment is indented when it doesn't need to be?